### PR TITLE
Add Billboard component

### PR DESCRIPTION
**Pull request by lndgalante**
- [x] Adds BaseUI and `gatsby-plugin-styletron` in order to work
- [x] Adds `<Billboard />` component that displays billboard movies and can be filtered by `cinemaId`
- [x] Removes and replace several components for BaseUI built-ins i.e Typography

Demo: https://vimeo.com/user97979398/review/382384425/ef2d981930
Deploy preview: https://deploy-preview-3--estrenos-web.netlify.com

### CODE REVIEW

File: src/components/ui/ScrollableSelect/index.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    >
+      {item.name}
+    </ListItem>
+  );
+};
+
+const VirtualDropdown = React.forwardRef((props, ref) => {
+  const children = React.Children.toArray(props.children);
+
+  // if (!children[0] || !children[0].props.item) {
```
Comment (hdf1996) [inline src/components/ui/ScrollableSelect/index.js]: En general prefiero borrar comments si no los estoy usando (sobre todo porque igual quedan en git si los llego a necesitar), yo diria de borrarlo!

**Review by hdf1996 — APPROVED** [review]
Un solo comment re simple, igual feel free to merge por mi parte

File: src/components/ui/ScrollableSelect/index.js (hunk 1 above)
Comment (lndgalante) [reply src/components/ui/ScrollableSelect/index.js]: Gracias @hdf1986 !

File: .gitignore [hunk 2]
```diff
@@ -62,8 +62,10 @@ public
 .DS_Store
 
 # Yarn
+yarn.lock
```
Comment (goncy) [inline .gitignore:65]: No se deberia commitear este file?

**Pull request merged**